scripts/setupvars/setupvars.bat

```diff
@@ -93,14 +99,14 @@ if "%pyversion_major%" equ "%PYTHON_VERSION_MAJOR%" (
 
 if not "%check_pyversion%"=="true" (
    echo Unsupported Python version. Please install one of Python %PYTHON_VERSION_MAJOR%.%MIN_REQUIRED_PYTHON_VERSION_MINOR% - %PYTHON_VERSION_MAJOR%.%MAX_SUPPORTED_PYTHON_VERSION_MINOR% ^(64-bit^) from https://www.python.org/downloads/
-   exit /B 1
+   exit /B 0
 )
 
 :: Check Python bitness
 python -c "import sys; print(64 if sys.maxsize > 2**32 else 32)" 2 > NUL
 if errorlevel 1 (
-   echo Error^: Error during installed Python bitness detection
-   exit /B 1
+   echo Warning^: Error during installed Python bitness detection
```

Comment:
for Linux we have: `64 bitness for Python $python_version is required`
So, no words like `error` which can confuse users

Reply:
fixed